docs/resiliency-checklist.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Resiliency Checklist
+resourceType: Documentation
+personas: 
+  - Developer
+tags:
+  - openshift
+  - ha
+  - high availability
+  - resilient
+  - devops
+description: How to design your application to have as much uptime as possible.
+---
+
+# Resiliency Checklist
+
+One of the most amazing benefits of working on the Openshift Platform is that, when your application has been designed with a few key ideas in mind, you can avoid many of the regular outages that are almost unavoidable on legacy infrastructure.
```

Comment:
Line breaks get eaten by markdown unless its is a double linebreak. So I would suggest these three lines should either be merged to one, or a blank line should be between them.

Reply:
oh i know, i tend to stick frivolous linebreaks into md files because it's easier for me to read when i'm actually doing the writing. i can get rid of them, though i'm not sure it matters?

Reply:
OK, I accept your frivolous linebreaks 😛